File: Unidad 2-Estructuras de Control/Ejercicio10.cs
//Equipo4
//Fecha 04/02/2020
//Autor: José Mirón
//Programa que recibe un numero N y da los primeros N multiplos de 7
//Entrada: Numero entero positivo
//Salida: N multiplos enteros positivos de 7
using System;

namespace Contador
{
    class Program
    {
        static void Main(string[] args)
        {
            //Entrada
            int n;
            n =Convert.ToInt32(Console.ReadLine());//valor del numero de multiplos deseados

            //Proceso
            for (int i = 7; i <= (7*n); i += 7)//inicio en cero, 7 multiplicado por el valor de n, incrementador
            {
                //Salida
                Console.WriteLine(i);//Impresion de los multiplos de 7 segun n
            }
            Console.ReadLine();
        }
    }
}

File: Unidad 2-Estructuras de Control/Ejercicio10.c
//Equipo4
//Autor:Jose Miron
//Programa que reciba un numero N y te de los primeros N multiplos de 7
//Entrada: Numero entero
//Salida: Multiplos de 7
#include <stdio.h>

int main() 
{   //Entrada
    int numMulti; //Entrada
    scanf("%d", &numMulti);//Se lee numero de multiplos
    
    //Proceso
    for(int i=7; i <=(7*numMulti); i+=7)//Ciclo para limite N
    { //Salida, impresion de N multiplos
        printf("%d \n", i);    
    }
return 0;
}
